index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <script  src="js/index.js"></script>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
```

Comment:
Kindly customize your project tittle to show what your project is all about.

Reply:
- [ ] Polite, clear comment explaining why change is required 🥳